tests/sql/parsing/queries/tpcds/test_tpcds.py

```diff
@@ -1,235 +1,390 @@
 """
 test parsing tpcds queries into DJ ASTs
 """
+import re
+
+# mypy: ignore-errors
+# pylint: skip-file
+from difflib import SequenceMatcher
 
 import pytest
+import sqlparse
+
+from dj.sql.parsing.backends.antlr4 import parse, parse_statement
+
+ansi_tpcds_files = [
```

Comment:
I can't see these files / variable used anywhere. What am I missing?

Reply:
Woah, thanks for catching this! This list just needed to be added to the parametrized test, pushed a commit to update that 👍